symbolic-demangle/vendor/swift/update.py

```diff
@@ -104,7 +104,7 @@ def main():
 
     print("> Cleaning up previous import")
     for entry in os.listdir(vendor_dir):
-        if os.path.isdir(entry):
+        if os.path.isdir(entry) and not workspace_dir.startswith(os.path.join(vendor_dir, entry)):
```

Comment:
guess what happened, lol…

Reply:
Well, yeah. You shouldn't check out swift inside of this folder, but I guess there's no harm in doing so.